src/main/java/com/_up/megastore/services/implementations/SizeService.java

```diff
@@ -46,14 +46,13 @@ public SizeResponse updateSize(UUID sizeId,UpdateSizeRequest updateSizeRequest){
 
     @Override
     public SizeResponse restoreSize(UUID sizeId){
-        validateSizeForRestoration(sizeId);
         Size size = findSizeByIdOrThrowException(sizeId);
+        validateSizeForRestoration(size, sizeId);
```

Comment:
Esto debería quedar así
```suggestion
        validateSizeForRestoration(size);
```